src/model_instance.rs

```diff
@@ -447,9 +447,9 @@ impl ModelInstance {
 			}
 		}
 
-		for (hyper_name, hyper_value) in replacement_hyperparam_ids.into_iter() {
-			println!("Warning! Updated hyperparameter {} to value {}", hyper_name, hyper_value);
-		}
+		// for (hyper_name, hyper_value) in replacement_hyperparam_ids.into_iter() {
```

Comment:
i think its a bad idea to remove this log. we have to have visibility if we update the HP

Reply:
Only commented it for this version; however, this does not seem to be the part responsible for most prints indeed, re-added.